### PR TITLE
Antora Dark Mode Enhancements and TOC Toggle Fixes

### PR DESCRIPTION
**Pull request by julioest**
- Improved dark mode styling for Antora with new variables and refined elements (code, quotes, tables)
- Added Asciidoctor TOC toggle and fixed table overflow issues
- Corrected font URL and optimized global styles

### CODE REVIEW

File: boostlook.css [hunk 1]
```diff
@@ -24,36 +24,96 @@
...
-  --bl-background: rgb(255, 255, 255);
-  --bl-card-background-color: rgb(255, 255, 255);
-  --bl-cyan-color: rgb(0, 90, 156);
+  --light-bl-pasteboard-color: rgb(209, 213, 219);
+  --light-bl-pre-background: rgb(248, 248, 248);
+  --light-bl-table-border-color: rgb(232, 232, 232);
+  --light-bl-table-head-background: rgb(248, 248, 248);
+  --light-bl-background: rgb(255, 255, 255);
+  --light-bl-card-background-color: rgb(255, 255, 255);
+  --light-bl-cyan-color: rgb(0, 90, 156);
```
Comment (rbbeeston) [inline boostlook.css:44]: why are there 2 of the below?

**Review by rbbeeston — APPROVED** [review]
one minor nit, other than that it looks good!

File: boostlook.css [hunk 2]
```diff
@@ -24,36 +24,96 @@
...
+  --light-bl-quote-background: rgb(250, 250, 250);
+  --light-bl-quote-background: rgb(255, 255, 255);
+  --light-bl-code-background: rgb(255, 255, 255);
```
Comment (rbbeeston) [inline boostlook.css:47]: why are there 2 of --light-bl-quote-background?

Comment (julioest) [reply boostlook.css:47]: Good catch! Per our discussion, I'll fix this in the next one

**Comment by julioest** [comment]
@vinniefalco Please merge this when you can

**Pull request merged**